x-pack/plugins/security_solution/server/lib/risk_engine/tasks/risk_scoring_task.ts

```diff
@@ -75,14 +74,12 @@ export const registerRiskScoringTask = ({
       // own purposes.  It appears the client is using ILM, and this won't work
```

Comment:
This whole comment block is related to the `dataStreamAdapter` below; you can remove it as well 👍